RailwayBooking.java

```diff
@@ -97,8 +101,45 @@ public boolean toLogin(String username, String password) {
         }
         return false;
     }
+    
+    public boolean isAdmin(String username, String password) {
+        for (Admin a : db.getAdmins()) {
+            if (a.getAdminName().equals(username) && a.getAdminPassword().equals(password)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public void toAddNewTrain( String nameOfTheTrain, HashMap<String, Integer> interMediateStation,int availableSeats, String dateOfTrain) {
+        Train newTrain = new Train(nameOfTheTrain,interMediateStation,availableSeats,dateOfTrain);
+        db.trains.add(newTrain);
+    }
 
-    private String generateTicketKey(String destination, ClassType classType, int seatNumber, BerthPreference berthPreference) {
+    public void toDeleteTrain(String nameOfTheTrain) {
```

Comment:
if a function does not return anything... function name should be a verb... so it should be deleteTrain and addTrain

Reply:
function name was changed